evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -188,6 +188,23 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate "Copy" %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+            <confirmation-modal type="submit" id="exam_creation_modal" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                <span slot="action-text">{% translate 'Create exam evaluation' %}</span>
+                <span slot="question">
+                    {% blocktranslate %}
```

Comment:
We should go with `blocktranslate trimmed`, otherwise the whitespace will be included in the string for the translation